presto-product-tests/pom.xml

```diff
@@ -18,6 +18,44 @@
         <scala.version>2.12.2</scala.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>3.14.0</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <profiles>
+        <profile>
+            <id>java17</id>
```

Comment:
Do we need this if the base pom already has this too ?

Reply:
We should verify the lists are exactly the same. If they are, we shouldn't need it